frontend/src/metabase-types/store/embedding-data-picker.ts

```diff
@@ -2,12 +2,19 @@ export interface EmbeddingDataPickerState {
   entityTypes: EmbeddingEntityType[];
 }
 
+// Duplicate the type instead, so we see this type name rather than `FullAppEmbeddingEntityType` if we do type EmbeddingEntityType = FullAppEmbeddingEntityType
+export type EmbeddingEntityType = "model" | "table" | "question";
+
 /**
  * `question` only works on multi-stage data picker, not the simple data picker.
  * The reason being that we want to streamline user experience for simple embedding
  * use cases, but `question` was later added to support users who are used to
  * selecting Saved questions in interactive embedding, so this is special case.
  */
-export type EmbeddingEntityType = "model" | "table" | "question";
+export type FullAppEmbeddingEntityType = "model" | "table" | "question";
+
+// `question` is only applicable to the staged picker, which is rarely used in SDK unless
```

Comment:
this originally was a JS doc comment with `@internal` but it messes up our doc and our [package generation](https://github.com/metabase/metabase/actions/runs/20506458351/job/58922807174?pr=67534#step:5:389).

What I want for this comment is to not be included in the doc, but keep the type in the doc. Using `@internal` or `@ignore` will just exclude it from the doc which isn't what we want.